dhis-2/pom.xml

```diff
@@ -35,6 +35,7 @@
     <module>dhis-support</module>
     <module>dhis-web-api</module>
     <module>dhis-web-embedded-jetty</module>
+    <module>dhis-support/dhis-support-test-json</module>
```

Comment:
We have an aggregate POM file in `dhis-support` folder which this module build instruction fits better.

Reply:
If I remember correctly I had it there first and ran into some problems. But now can't see why this would not work - I think my problems were more related to `<rootDir>` not being correctly set. Changed it and it seems to work fine.